scripts/unit-jest.sh

```diff
@@ -7,7 +7,7 @@ export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/unit"
 echo $TEST_SUITE_TYPE > $TEST_SUITE_TYPE_FILE
 echo $TEST_RESULT_FILE_DIR > $TEST_RESULT_FILE_DIR_FILE
 
-if ! yarn test --type jest --runInBand; then
```

Comment:
was `--runInBand` causing issues? This is actually good though since it should run tests concurrently (faster overall)

Reply:
Yea, using the `--runInBand` flag caused the test to timeout (it took > 30mins to actually complete). When I looked up what the flag was meant for I realized we did not need it in bacon (only during debuggin). This decreased the time to ~10 mins